src/configuration.ts

```diff
@@ -25,7 +25,7 @@ import { arrayify, hasOwnProperty, stripComments } from "./utils";
 
 export interface IConfigurationFile {
     /**
-     * The severity that is applied to rules in _this_ config with `severity === "default"`.
+     * The severity that is applied to rules in this and _extended_ configs with `severity === "default"`.
```

Comment:
This comment should probably be more verbose: `The severity that is applied to rules in this config file as well as rules in any inherited config file which have their severity set to "default"`